boefjes/boefjes/plugins/kat_fierce/normalize.py

```diff
@@ -17,7 +17,11 @@ def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterator[OOI
     yield internet
 
     for _, subdomain in results["subdomains"].items():
-        host = Hostname(name=subdomain["url"], network=internet.reference)
+        if subdomain["url"][-1:] == ".":
```

Comment:
```suggestion
        if subdomain["url"].endswith("."):
```

Reply:
Another (cleaner approach) would be:
```python3
...
subdomain = subdomain["url"].rstrip(".")
host = Hostname(name=subdomain, network=internet.reference)
...
```

Reply:
Thanks for the feedback @ammar92 I implemented these changes, the suggested as well as the cleaner approach but kept the if-else statement there.